python/tvm/auto_scheduler/relay_integration.py

```diff
@@ -66,9 +67,12 @@ def call_all_topi_funcs(mod, params, target, opt_level=3):
         if params:
             compiler.set_params(params)
         mod = tvm.IRModule.from_expr(mod) if isinstance(mod, relay.Function) else mod
-        compiler.lower(mod, target)
-
-    autotvm.GLOBAL_SCOPE.silent = old_autotvm_silent
+        try:
+            compiler.lower(mod, target)
+        except Exception: # pylint: disable=broad-except
```

Comment:
Do we want to silently eat any error? For example, this will prevent Ctrl-C from working. It might be better to check for TVMError

Reply:
Ah, I thought the ffi ate the type of all exceptions. I see other usages of using TVMError and it works this way so is preferable.